docs/google-cloud.md

```diff
@@ -14,7 +22,9 @@ You can enable APIs from the command line with:
 gcloud services enable compute container containerregistry.googleapis.com
```

Comment:
I think here we want to also include:
 - compute.googleapis.com
 - container.googleapis.com

Reply:
Did I add it in the correct way?

Reply:
I think so :)